pf/internal/muxer/muxer.go

```diff
@@ -253,4 +265,8 @@ func (p *simpleSchemaProvider) DataSourcesMap() shim.ResourceMap {
 	return p.dataSources
 }
 
+func (p *simpleSchemaProvider) DetailedSchemaDump() []byte {
+	panic("Unsupported")
```

Comment:
Instead of panicking, can we just return nil. That way we can leave actual panics uncaught.
```suggestion
	return nil
```